### DISCUSSION TITLE
[Suggestion] Allow placing bowls as decorations

### DISCUSSION THREAD

**Discussion by MarioSMB** [vectorwing/FarmersDelight#531] [Ideas]
After a big feast a dirty emptied bowl is left in its place, it would be cool to be able to place clean bowls down for decorative purposes!

**Comment by RaymondBlaze** [comment]
It looks like a good idea at the first sight, but it may turn out troublesome.

I don't know if you have played Botania, it has implemented a feature of "when players use Glass Bottles in the End , it will turn into an Ender Air Bottle", and thus made drinking potion in the End become quite annoying. A placing bowl feature is much like this.

A large amount for meals uses bowl as the container, which means a bowl will be replace the meal item on ate, if there's only 1 meal item in your hand. And here's the problem: the player right click and hold to eat, it's very likely that they are still holding the right click on finish eating. In that case, if a "placing bowl" feature is implemented, the leftover bowl will be placed onto the ground, which is unexpected.

A solution could be: Adding a plate block/item and use it for decoration only.

**Comment by MarioSMB** [comment]
That is a very detailed examination of the issue, thank you. I’d definitely hate to see the sweet berry situation repeated!

A separate item for a clean dish sounds like an elegant solution to me. It may be able to serve some functional use later on, but even as a purely decorative block it would be a nice touch for kitchen aesthetics!
